fairlearn/datasets/_synthetic.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+import numpy as np
+from sklearn.datasets import make_classification
+from sklearn.utils import check_random_state
+
+
+def make_synthetic_dataset(feature_config=None, n_features=20, n_informative=4, random_state=None):
```

Comment:
Can this API reasonably be extended to multiple sensitive features in the future? If not, I'd suggest tweaking the name to reflect that it can only generate for a single sensitive feature.

Reply:
Yes and it will be. I wrote sort of a proposal for what that might look like in another comment.